api-samples/tabs/inspector/tabs_api.js

```diff
@@ -0,0 +1,412 @@
+let tabs = {};
+let tabIds = [];
+
+let focusedWindowId = undefined;
+let currentWindowId = undefined;
+
+async function bootstrap() {
+  const currentWindow = await chrome.windows.getCurrent();
+  currentWindowId = currentWindow.id;
+  const focusedWindow = await chrome.windows.getLastFocused();
+  focusedWindowId = focusedWindow.id;
+  loadWindowList();
+}
+
+function isInt(i) {
+  return typeof i == 'number' && !(i % 1) && !isNaN(i);
+}
+
+const windowTemplate = document.getElementById('windowItem').content;
+const tabTemplate = document.getElementById('tabItem').content;
+
+async function loadWindowList() {
+  const windowList = await chrome.windows.getAll({ populate: true });
+  tabs = {};
+  tabIds = [];
+  for (let window of windowList) {
+    for (let tab of window.tabs) {
+      tabIds.push(tabIds);
```

Comment:
I think this might be a bug?

```suggestion
      tabIds.push(tab.id);
```